protected/humhub/modules/notification/widgets/views/notificationSettingsForm.php

```diff
@@ -39,8 +39,10 @@
                     </td>
                     <?php foreach ($model->targets() as $target): ?>
                         <td class="text-center">
-                            <?php $disabled = !$target->isEditable($model->user) || $category->isFixedSetting($target) ?>
-                            <?= Html::checkbox($model->getSettingFormname($category, $target), $target->isCategoryEnabled($category, $model->user), ['style' => 'margin:0px;', 'disabled' => $disabled]) ?>
+                            <?php if(! ($target->id == 'web' && $category->id == 'admin_user_updates')): ?>
```

Comment:
You should be able to use the isVisible(User $user) function instead of this if statement. Just filter out all non admins in the isVisible function.